convert2rhel/utils.py

```diff
@@ -702,44 +704,51 @@ def is_dir_empty(path):
             os.rmdir(path)
 
 
-def hide_secrets(args, secret_args=frozenset(("--password", "--activationkey", "--token", "-p", "-k"))):
+def hide_secrets(
+    args, secret_options=frozenset(("--username", "--password", "--activationkey", "--org", "-u", "-p", "-k", "-o"))
+):
     """
-    Replace secret values with asterisks.
+    Replace secret values passed through a command line with asterisks.
 
-    This function takes a list of arguments which will be passed
-    in a transformation process where we will replace any secret values
-    with an fixed size of asterisks (*) and returns a new list containing
-     the arguments with this transformation.
+    This function takes a list of command line arguments and returns a new list containing where any secret value is
+    replaced with a fixed size of asterisks (*).
 
-    :arg args: An argument list which may contain secret values.
+    Terminology:
+     - an argument is one of whitespace delimited strings of an executed cli command
+       Example: `ls -a -w 10 dir` ... four arguments (-a, -w, 10, dir)
+     - an option is a subset of arguments that start with - or -- and modify the behavior of a cli command
+       Example: `ls -a -w 10 dir` ... two options (-a, -w)
+     - an option parameter is the value following an aption if the option requires a value
+       Example: `ls -a -w 10 dir` ... one option parameter (10)
```

Comment:
If `-a` were a boolean or a flag of some sort, then having the `-a` in there should be: 

```suggestion
       Example: `ls -a -w 10 dir` ... one option parameter (True, 10)
```

of course, considering argparse in this comment.

Reply:
not an suggestion to change, though. Just a comment that I thought it would make sense to add here.

Reply:
In the terminology I explain how we are to name cli arguments. The "True" value is not an argument passed through CLI. It might be inferred by the CLI utility internally, based on the programming language the utility is written in, or the library used for parsing the CLI arguments.

I'm changing:
```
- an option parameter is the value following an aption if the option requires a value
```
to
```
- an option parameter is the argument following an aption if the option requires a value
```
to make it a bit clearer.